platformio.ini

```diff
@@ -195,325 +173,295 @@ build_flags =
   -DMIMETYPE_MINIMAL
   ; other special-purpose framework flags (see https://docs.platformio.org/en/latest/platforms/espressif8266.html)
   ; decrease code cache size and increase IRAM to fit all pixel functions
-  -D PIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48 ;; in case of linker errors like "section `.text1' will not fit in region `iram1_0_seg'"
-  ; -D PIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48_SECHEAP_SHARED ;; (experimental) adds some extra heap, but may cause slowdown
+  -DPIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48 ;; in case of linker errors like "section `.text1' will not fit in region `iram1_0_seg'"
+  ; -DPIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48_SECHEAP_SHARED ;; (experimental) adds some extra heap, but may cause slowdown
 
 lib_deps =
-  #https://github.com/lorol/LITTLEFS.git
+  ${env.lib_deps}
   ESPAsyncTCP @ 1.2.2
   ESPAsyncUDP
-  ${env.lib_deps}
 
 [esp32]
-#platform = https://github.com/tasmota/platform-espressif32/releases/download/v2.0.2.3/platform-espressif32-2.0.2.3.zip
-platform = espressif32@3.5.0
-platform_packages = framework-arduinoespressif32 @ https://github.com/Aircoookie/arduino-esp32.git#1.0.6.4
-build_flags = -g
+platform = espressif32@6.7.0
```

Comment:
Sorry but the ESP-IDF version included in the "standard" platform 6.7.0 has known bugs, we cannot use it.
The latest usable arduino-esp32 versions are 2.0.9 (well tested) and 2.0.14 (seems to work). Everything that came after 2.0.14 is hot garbage.

Reply:
espressif32@3.5.0 is Arduino core 1.0.6
A change to espressif32@6.7.0 is a change to core 2.0.16
Major version change!! Btw. 2.0.16 has bugs. Only latest version in branch release/v2.x in repo should be used (not released).
Core 2.0.14 is working not bad at all, if a release version is needed